scripts/avail-ext-doc/list-template.md

```diff
@@ -17,10 +17,10 @@ This article is a complete list of the available extensions for the Azure CLI wh
 
 The list of extensions is also available  from the CLI. To get it, run [az extension list-available](/cli/azure/extension?view=azure-cli-latest#az-extension-list-available):
 
-```azurecli
+```azurecli-interactive
```

Comment:
@sptramer can you give some insight on this change?

Reply:
This enables the "try it" button on the code sample, which is a request from the documentation LT to enable across the site.